src/_includes/docs/swift-package-manager/how-to-enable-disable.md

```diff
@@ -0,0 +1,77 @@
+## How to turn on Swift Package Manager
+
+Flutter's Swift Package Manager support is turned off by default.
+To turn it on:
+
+1. Switch to Flutter's `main` channel:
+
+   ```sh
+   flutter channel main
+   ```
+
+2. Make sure you have the latest Flutter SDK:
+
+   ```sh
+   flutter upgrade
+   ```
+
+3. Turn on the Swift Package Manager feature:
+
+   ```sh
+   flutter config --enable-swift-package-manager
+   ```
+
+Running an app using the Flutter CLI migrates it to add Swift Package Manager
```

Comment:
nit: this sentence was a bit hard to follow.

every time I run the app, the app is migrated?
what is "it" in the above? the CLI or my app? (I assume the app, but would be good to spell out)

Reply:
Good feedback!

> every time I run the app, the app is migrated?

Yup that's correct. Each time you build your app, the tool will check if it needs to modify the app.

Reply:
Updated to the suggestion here: https://github.com/flutter/website/pull/10827#discussion_r1678456670